presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -3125,6 +3185,96 @@ protected void doTestTransactionDeleteInsert(HiveStorageFormat storageFormat, Sc
         }
     }
 
+    private void doTestMismatchSchemaTable(SchemaTableName schemaTableName, HiveStorageFormat storageFormat)
```

Comment:
This method should be placed directly below `testMismatchSchemaTable`, so they can viewed together